### PR TITLE
Features

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/trabajopolis/ui/pages/JobOffer.java [hunk 1]
```diff
@@ -66,17 +71,30 @@ public String getMessageCity() {
...
+        String time = getMessageTime().substring(11);
+        String numeralDate = TrabajopolisUtils.trasnformDateToNumeral(time);
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy");
+        try {
+            Date fechaInicial = dateFormat.parse(numeralDate);
+            Calendar cal = Calendar.getInstance();
+            Date fechaFinal = dateFormat.parse(dateFormat.format(cal.getTime()));
+            days = (int) ((fechaFinal.getTime() - fechaInicial.getTime()) / 86400000);
+        } catch (ParseException e) {
+            e.printStackTrace();
```
Comment (MadayAlcala) [inline src/main/java/trabajopolis/ui/pages/JobOffer.java]: Please use a throw instead a printstacktrace.

**Review by MadayAlcala — COMMENTED** [review]
Please see my comments.

File: src/main/java/trabajopolis/ui/pages/JobOffer.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/trabajopolis/ui/pages/JobOffer.java]: done

**Review by MadayAlcala — APPROVED** [review]
great!

**Pull request merged**